binding/react-native/README.md

```diff
@@ -145,10 +145,18 @@ inferenceCallback(object){
 }
 ```
 
-You can override also the default Rhino model file and/or the inference sensitivity.  You can set `requireEndpoint` parameter to
-false if you do not wish to wait for silence before Rhino infers context. There is also an optional `processErrorCallback`
+You can override also the default Rhino model file and/or the inference sensitivity. There is also an optional `processErrorCallback`
 that is called if there is a problem encountered while processing audio.
 
+`endpointDurationSec` indicates how much silence (in seconds) Rhino will wait for before marking the end of a spoken command. A lower endpoint
+duration reduces delay and improves responsiveness. A higher endpoint duration assures Rhino doesn't return inference
+pre-emptively in case the user pauses before finishing the request.
+
+`requireEndpoint` is the parameter which indicates if Rhino should wait for a silence before inferring context.
+If set to `true`, Rhino requires an endpoint (a chunk of silence) after the spoken command. If set to `false`,
+Rhino tries to detect silence, but if it cannot, it still will provide inference regardless. Set
+to `false` only if operating in an environment with overlapping speech (e.g. people talking in the background).
```

Comment:
This may be too much information for the README. These features are only useful to a small subset of our users.

Maybe something more like:
"There are two optional parameters for controlling when and how Rhino returns an inference. `endpointDurationSec` sets how much silence is required after a spoken command and `requireEndpoint` indicates whether Rhino should wait for silence before returning an inference."